backend/question/src/controller/search-controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from 'express';
+import { searchQuestionsByTitleService } from '../services/get/index';
+
+export const searchQuestionsByTitle = async (req: Request, res: Response): Promise<Response> => {
+  const { title } = req.query;
+  const page = parseInt(req.query.page as string) || 1;
+  const limit = parseInt(req.query.limit as string) || 10;
+
+  if (!title) {
```

Comment:
if no title is given, I'd suggest just returning all questions. Because in the frontend, we shouldn't give an error if the search box is empty, since it is acceptable for the search box to be empty